kyuubi-server/web-ui/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "kyuubi-ui",
+  "private": true,
+  "version": "1.6.0-SNAPSHOT",
+  "type": "module",
+  "scripts": {
+    "dev": "vite --port 8080",
```

Comment:
let switch to another port like 9090, 9 is the lucky number for kyuubi

Reply:
sure~